app/verify_workflow.go

```diff
@@ -87,6 +87,11 @@ func verifyScorecardWorkflow(workflowContent string) error {
 	}
 
 	// Verify that the workflow runs on ubuntu and nothing else.
+	// This would never happen because it will fail the linting.
+	/*
+		errors during actionlint.Parse: [:16:3: "runs-on" section is missing in job "analysis" [syntax-check]]
+	*/
+	// TODO("naveen") :- Remove this check
 	if scorecardJob.RunsOn == nil {
 		return fmt.Errorf("%w", errScorecardJobRunsOn)
 	}
```

Comment:
Since you already have the test for this we can remove it safely in this PR itself if the test passes with this check removed.

Reply:
I have removed it.Thanks